app/components/forms/admin/content/pages-form.js

```diff
@@ -0,0 +1,78 @@
+import Ember from 'ember';
+import FormMixin from 'open-event-frontend/mixins/form';
+
+const { Component } = Ember;
+
+export default Component.extend(FormMixin, {
+
+  getValidationRules() {
+    return {
+      inline : true,
+      delay  : false,
+      on     : 'blur',
+      fields : {
+        name: {
+          identifier : 'name',
+          rules      : [
+            {
+              type   : 'empty',
+              prompt : this.i18n.t('Please enter a name')
+            }
+          ]
+        },
+        title: {
+          identifier : 'title',
+          rules      : [
+            {
+              type   : 'empty',
+              prompt : this.i18n.t('Please enter a title')
+            }
+          ]
+        },
+        url: {
+          identifier : 'url',
+          rules      : [
+            {
+              type   : 'empty',
+              prompt : this.i18n.t('Please enter an url')
+            }
+          ]
+        },
+        place: {
+          identifier : 'place',
+          rules      : [
+            {
+              type   : 'empty',
+              prompt : this.i18n.t('Please select a place')
+            }
+          ]
+        },
+        position: {
+          identifier : 'position',
+          rules      : [
+            {
+              type   : 'empty',
+              prompt : this.i18n.t('Please enter a position')
```

Comment:
This is not in accordance with the new localisation standard, please change it to `l10n`